programs/tbench/run

```diff
@@ -1,14 +1,25 @@
 #!/bin/sh
 # - runtime
 # - nr_threads
+# - iterations
 
 ## TBENCH produces the network load to simulate the network part of
 ## the load of the commecial Netbench benchmark.  Unlike Netbench,
 ## it doesn't require a lab of Windows PCs:
+. $LKP_SRC/lib/reproduce-log.sh
 
 [ -n "$direct_server_ips" ] && server=$direct_server_ips
 [ -z "$server" ] && server=127.0.0.1
 [ -n "$runtime" ] || runtime=600
-server=${server%% *}
+[ -n "$iterations" ] || iterations=1
+
+log_cmd command -v tbench || die "tbench not found"
```

Comment:
Here can consider to use has_cmd function

Reply:
Sure will do